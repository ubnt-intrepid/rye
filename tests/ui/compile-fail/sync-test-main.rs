fn main() {}

#[rye::test_main]
fn test_main() {}
