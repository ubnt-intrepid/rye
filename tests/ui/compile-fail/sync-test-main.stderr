error: non-async function is not accepted
 --> $DIR/sync-test-main.rs:4:1
  |
4 | fn test_main() {}
  | ^^^^^^^^^^^^^^^^^
